### ISSUE TITLE
Model objects handling 

### ISSUE THREAD

**Issue by MDM-Solution** [aarav-tech/ems#1]
Hi sir.... Can I display the questions as per user.... Am trying to ....because all questions are referring for all user can it be restricted by manager id

**Comment by aarav-tech** [comment]
Yes. You can use filter to get questions by manager id if they are associated using foreign key.

**Comment by PUSHPENDERSINGH786** [comment]
Hi Sir,

Thanks for your reply sir and also for online educations videos.i learned a
lots of thinks from your shared videos.
